README.md

```diff
@@ -2,6 +2,16 @@
 
 Web application for Checkers Board Game. The AI we use is the AlphaGo Zero algorithm implemented by DeepMind.
 
+
+## Project Description
+
+Alphazero is a breakthrough application of reinforcement learning in board games. Thus, this work will try implementing RL to checkers game. First, a user interface is made for multiplayer checkers game. We also created a checkers bot with Minimax algorithm to serve as training agent for our model. Monte Carlo tree search along with policy and value networks will be integrated to our model in the next stage. 
```

Comment:
- `Minimax algorithm to serve as training agent for our model` should be "_a validating agent_"?
-  `will be integrated to our model` "_to our project_" maybe appropriate

@aungmcs @aungpaing98 Please check this project description. Any suggestions?